client/my-sites/stats/activity-log/index.jsx

```diff
@@ -536,24 +527,58 @@ class ActivityLog extends Component {
 
 		const restoreConfirmDialog = requestedRestoreActivity && (
 			<ActivityLogConfirmDialog
-				applySiteOffset={ this.applySiteOffset }
 				key="activity-rewind-dialog"
-				onClose={ this.handleCloseDialog }
-				onConfirm={ this.handleConfirmDialog }
-				timestamp={ requestedRestoreActivity.activityTs }
-			/>
+				confirmTitle={ translate( 'Confirm Rewind' ) }
+				onClose={ this.dismissRestore }
+				onConfirm={ this.confirmRestore }
+				title={ translate( 'Rewind Site' ) }
+			>
+				{ translate(
+					'This is the selected point for your site Rewind. ' +
+						'Are you sure you want to rewind your site back to {{b}}%(time)s{{/b}}?',
+					{
+						args: {
+							time: this.applySiteOffset(
+								moment.utc( requestedRestoreActivity.activityTs )
+							).format( 'LLL' ),
+						},
+						components: { b: <b /> },
+					}
+				) }
```

Comment:
I don't see why we have to reveal markup details to translators. We could hide the markup details to translators like:
```
{ translate(
	'We will build a downloadable backup of your site at {{time/}}. ' +
		'You will get a notification when the backup is ready to download.',
	{
		components: { time: <b>{ this.applySiteOffset( moment.utc( requestedBackup.activityTs ) ).format( 'LLL' ) }</b> },
	}
) }
```
Related p1510255067000548-slack-calypso

Reply:
Sure, sounds fine. This is not a change I introduced. How about we update in a separate PR?

Reply:
Since you're already doing this large refactor, I think it's perfectly ok to do this small change here and fix all at once.

Reply:
changed in 3e5c68d41